src/System.Device.Gpio/System/Device/Gpio/Processor.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace System.Device.Gpio.System.Device.Gpio
```

Comment:
Here: This namespace name causes confusion.

Reply:
Oh my God! Such a thing has never happened to me. Thank you @pgrawehr